core/platform/auth/firebase_auth_services_test.py

```diff
@@ -190,7 +208,11 @@ def delete_user(self, uid):
             raise firebase_auth.UserNotFoundError('%s not found' % uid)
         del self._users_by_uid[uid]
 
-    def delete_users(self, uids, force_delete=False):
+    def delete_users(
+            self,
+            uids: List[Text],
+            force_delete: bool = False
+    ) -> 'firebase_auth.BatchDeleteAccountsResponse':
```

Comment:
Due to some unknown reason, `firebase_auth.BatchDeleteAccountsResponse` is not recognized here and throws firebase_auth has no attribute BatchDeleteAccountsResponse. But [here](https://github.com/oppia/oppia/blob/7c70f79de3ae649ef3c6114ffbb67c6c66f186d8/core/platform/auth/firebase_auth_services_test.py#L825) it is detected. 
Even it is not available in [firebase_admin.auth](https://github.com/firebase/firebase-admin-python/blob/master/firebase_admin/auth.py) on their github repo.  But it is available in[ firebase_admin._user_mgt](https://github.com/firebase/firebase-admin-python/blob/0e35c9a6ce7bc4bd8034c5369e601cb440cff061/firebase_admin/_user_mgt.py#L436). But if I use this, linter throws accessing private variables is not allowed. So putting it in quotes seems the best option. Thanks!

Reply:
I think this will help out in future. Thanks!

Reply:
@hardikkat24 Can you share the error log here - it seems weird, because if we can use that function, and we also have stubs defined for it, what exactly is the problem then. @vojtechjelinek are you able to understand this behaviour?

Reply:
Not really, I imagine it can be related to some internal structure of firebase_admin where some components are loaded dynamically. But I don't think we need to bother with this, I think the long-term solution is to introduce `from __future__ import annotations` or move to Python 3.8 (which should be fairly easy compared to the Python 2 to Python 3 migration).

Reply:
>  Can you share the error log here
No error during mypy stubs because we have stubs. But error while running backend test. I agree with what @vojtechjelinek said. Thanks!

Reply:
> move to Python 3.8

@vojtechjelinek Are we going to do this soon?

Reply:
I don't know, we still need to finish the Python 3 migration (mainly Beam jobs) so there are other priorities. On the other hand, I don't really want to delay it so we should do it in the next few months. I also assume it will be quite simple and easy.